app/controllers/accounts_controller.rb

```diff
@@ -19,6 +19,13 @@ def show
       user: current_user,
       locked_for_session: pii_locked_for_session?(current_user),
     )
+    if session[:from_select_email_flow]
+      flash[:success] = t(
```

Comment:
I haven't tested, but i'd expect if we wanted the flash to be shown at the same time as the action currently being handled, we'd need `flash.now` rather than `flash`. Can you confirm?

```suggestion
      flash.now[:success] = t(
```

Reply:
Can we also add some test coverage?

Reply:
Looking it seems the flash[:success] still shows at the time of the action, but I dont mind doing flash.now to assure it does.

Reply:
I'm sorta wondering if what you're seeing is because of https://github.com/18F/identity-idp/pull/11443#discussion_r1828183556, since that particular flash would be assigned before the redirect. But unclear if it would also behave as expected when arriving from the sign-in page.

Reply:
I tested it when I had came from the sign in page and It had the same result.